inputCandidateGenerator.js

```diff
@@ -73,7 +73,7 @@ function sTreeGEN(terminalString, options)
     if(options.noMirrorImages){
       sTreeList = sTreeList.filter(x => !mirrorImages(x, sTreeList));
     }
-    if(options.noBarLevels){
+    if(!options.noBarLevels){
```

Comment:
This works, but it would be better to leave the logic in sTreeGEN() the same, and flip the value of noBarLevels before passing it in as an option. I was intending this change to be entirely on the interface. I just want to be sure that if nothing is specified for noBarLevels, sTreeGEN() creates binary branching trees rather than ternary branching trees.